### DISCUSSION TITLE
client-side caching

### DISCUSSION THREAD

**Discussion by hadez0528** [ApolloAuto/apollo#15549] [Q&A]
How do I implement client-side caching with Apollo Client?

**Comment by Stormer0528** [comment]
To implement client-side caching with Apollo Client, you can use the InMemoryCache as your cache implementation. When initializing the Apollo Client, you can specify the cache like this:

```
import { ApolloClient, InMemoryCache } from '@apollo/client';

const client = new ApolloClient({
  uri: 'https://your-graphql-endpoint.com/graphql',
  cache: new InMemoryCache(), // This initializes the InMemoryCache
});

```
